pkg/cli/admin/release/info.go

```diff
@@ -587,7 +596,17 @@ func findImageSpec(image *imageapi.ImageStream, tagName, imageName string) (stri
 	for _, tag := range image.Spec.Tags {
 		if tag.Name == tagName {
 			if tag.From != nil && tag.From.Kind == "DockerImage" && len(tag.From.Name) > 0 {
-				return tag.From.Name, nil
+				ref, err := imagereference.Parse(tag.From.Name)
+				if err != nil {
+					return "", err
+				}
+				preferred, err := imagereference.Parse(imageName)
+				if err != nil {
+					return "", err
+				}
+				preferred.ID = ref.ID
+				preferred.Tag = ref.Tag
+				return preferred.String(), nil
```

Comment:
there is a way this won't be necessary.. looking for it

Reply:
kind of removed it - moved to the `readReleaseImageReferences` in adm/release/image_mapper.go